src/System.Runtime/src/System/IO/FileStream.cs

```diff
@@ -0,0 +1,49 @@
+namespace System.IO {
+    public class FileStream : System.IO.Stream
+    {
+        [Obsolete("This constructor has been deprecated.  Please use new FileStream(SafeFileHandle handle, FileAccess access) instead.  http://go.microsoft.com/fwlink/?linkid=14202")]
```

Comment:
It helps with ILC.exe command not failing since it sees that xunit runner depends on this type and it can't find it. Xunit on the other hand, only depends on FileStream for writing the results.xml which will be disabled temporarily, so this stub is necessary for now.